releaseNotes/Microsoft.Featuremanagement.AspNetCore.md

```diff
@@ -2,6 +2,26 @@
 
 [Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
+## 4.0.0 - September 24, 2024
+
+### Enhancements
+
+* Adjusted the `.WithTargeting()` builder method to automatically add `HttpContextAccessor` if it's not already added.
+* Added `TargetingHttpContextMiddleware` which persists targeting context in the current activity. This is used when setting up [Telemetry](https://learn.microsoft.com/en-us/azure/azure-app-configuration/feature-management-dotnet-reference#telemetry).
+* Added support for variants in `FeatureTagHelper`. This allows MVC views to use the `<feature>` tag to conditionally render content based on whether a specific variant of a feature is assigned.
+
+    ``` HTML+Razor
+    <feature name="FeatureX" variant="Alpha">
+      <p>This can only be seen if variant 'Alpha' of 'FeatureX' is assigned.</p>
+    </feature>
+    ```
+
+    For more details on ASP.NET views and variants, see [here](https://learn.microsoft.com/en-us/azure/azure-app-configuration/feature-management-dotnet-reference#view).
+
```

Comment:
We should still keep the note, right?

Updated `Microsoft.FeatureManagement` reference to `4.0.0`...